givenergy_modbus/client/commands.py

```diff
@@ -147,6 +328,9 @@ def set_inverter_reboot() -> list[TransparentRequest]:
     """Restart the inverter."""
     return [WriteHoldingRegisterRequest(RegisterMap.REBOOT, 100)]
 
+def set_active_power_rate(target: int) -> list[TransparentRequest]:
+    """Set max inverter power rate"""
+    return [WriteHoldingRegisterRequest(RegisterMap.ACTIVE_POWER_RATE, target)]
```

Comment:
My plan was to prefer instead to use write_named_register('active_power_rate', target)
which could also do a validity check on the value automatically.

And then rather than having to enter loads of trivial setter methods, I was playing with having them created on the fly, in a similar way to the new inverter model. See https://github.com/divenal/givenergy-modbus-async/commit/05f543f0139741534c4bb322d6cee4bedc7313c1

However, there's now a complication...